src/components/TaskList.tsx

```diff
@@ -21,7 +22,11 @@ export function TaskList() {
       </Stack>
       {/* End: this should be deleted whenever you're comfortable */}
 
-      <Box bg={'orange'}>Task list may go here here</Box>
+      <Box bg={'blue.700'}>
+        <SimpleGrid columns={{base: 1, sm: 2, md: 3}} spacing={3} mx={{ base: 4, md: 20 }}>
+          {tasks?.map((task) => <TaskBox task={task}></TaskBox>)}
```

Comment:
If you'll check the browser console you'll see the `Warning: Each child in a list should have a unique "key" prop.` error in this component. You can find the explanation of this error in React [official docs](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key) but a quick fix would be
```suggestion
          {tasks?.map((task) => <TaskBox task={task} key={task.id}></TaskBox>)}
```